.github/workflows/deploy.yml

```
@@ -29,10 +29,10 @@ jobs:
         run: |
           for DIR in guides/doc-*; do
             pushd $DIR &>/dev/null
+            [[ -d images/ ]] || (echo "Required images/ directory missing in $DIR"; exit 1)
+            [[ -h images/common ]] || (echo "Required images/common symlink missing in $DIR"; exit 1)
```

This will fail early if we don't have these directories.